__tests__/cst/common.js

```diff
@@ -33,7 +33,9 @@ export const testSpec = (res, exp) => {
         ? 'strValue' in res
           ? res.strValue
           : res.rawValue
-        : res
+        : typeof res === 'string'
+          ? res
+          : res.char
```

Comment:
It's possible to be string here:

```
  ● 7.3.3. Plain Style › Example 7.10. Plain Characters

    expect(received).toBe(expected) // Object.is equality

    Expected: " Outside flow collection:"
    Received: undefined

    Difference:

      Comparing two different types of values. Expected string but received undefined.

      35 |           : res.rawValue
      36 |         : res.char
    > 37 |     expect(value).toBe(exp)
         |                   ^
      38 |   } else if (Array.isArray(exp)) {
      39 |     expect(res).toBeInstanceOf(Array)
      40 |     trace: 'test-array', exp
```

Reply:
Ah, you're, right, I'd forgotten how `testSpec` is used by the tests for comment values.